ztag/transforms/mongodb.py

```diff
@@ -0,0 +1,29 @@
+import copy
+from ztag.transform import ZGrab2Transform
+from ztag import protocols
+
+
+class MongoDBTransform(ZGrab2Transform):
+
+    name = "mongodb/banner"
+    port = None
+    protocol = protocols.MONGODB
+    subprotocol = protocols.MONGODB.BANNER
+
+    def __init__(self, *args, **kwargs):
+        super(MongoDBTransform, self).__init__(*args, **kwargs)
+
+    def _transform_object(self, obj):
+        zout = super(MongoDBTransform, self)._transform_object(obj)
+        results = self.get_scan_results(obj)
```

Comment:
@justinbastress Remind me what `get_scan_results` does again? I don't remember how we implemented the ZGrab2 transforms. Is it returning a dict or one of those weird dict wrapper objects that catch `KeyError` and `IndexError`?

Reply:
So `get_scan_results()` manages pulling out the results from e.g. `{"data":{"mongoldb":{"results":{...}}}}`; but the fact that we are overriding `_transform_object` rather than `transform_object` (or whatever the other version is) means that it gets wrapped to catch those errors.